### PR TITLE
feat: now problem description is a pure markdown file

### PR DESCRIPTION
**Pull request by notdp**
This `h2m.js` file is my webpack output file.

I have run the test in my local browser and it works!

I'm not sure if this way of adding can work on the chrome plugin.

### CODE REVIEW

File: scripts/leetcode.js [hunk 1]
```diff
@@ -298,6 +298,13 @@ function checkElem(elem) {
   return elem && elem.length > 0;
 }
 
+/* Convert html to markdown */
+import html2md from './h2m'
```
Comment (QasimWani) [inline scripts/leetcode.js:302]: have u experimented it on the chrome extension? i'm not familiar with the usage of libraries within chrome API

**Review by QasimWani — COMMENTED** [review]
looks good.  lmk your test results on the extensions itself and how much longer (approx.) does it take to push to github.

File: scripts/leetcode.js (hunk 1 above)
Comment (notdp) [reply scripts/leetcode.js:302]: sorry, I am not familiar with chrome extension.

This PR's change doesn't work on chrome extension.

I implemented this function in another branch in a better way, and tested it with my local chrome extension.

Comment (QasimWani) [reply scripts/leetcode.js:302]: right, i was referring to your local chrome extension. could u explain why it's better than the current approach, i.e. html parsing?

Comment (notdp) [reply scripts/leetcode.js:302]: Actually，I write leetcode problem notes based on a leet hub repository.
I made a turndown-based-converter that convert html text to md text.It actually does something I expected.
I think there may be a lot people would like to write their thinking about problem in leethub-generated file.

Comment (notdp) [reply scripts/leetcode.js:302]: Add:
A pure md file is more editable than normal html file.
If users want to write here than to read here,md file is a better choice.

**Pull request closed without merging**